nixos/modules/services/networking/keepalived/vrrp-instance-options.nix

```diff
@@ -24,7 +24,7 @@ with lib;
     virtualRouterId = mkOption {
       type = types.int;
```

Comment:
Shall we use `types.ints.between 1 255`

Reply:
I didn't know about `ints.between` but yeah, that seems wise, this would fail at eval time not runtime.